contents/views.py

```diff
@@ -1,3 +1,103 @@
 from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.serializers import ModelSerializer
+from rest_framework.views import APIView
+
+from contents.models import Contents_table
+
 
 # Create your views here.
+
+class ContentsSerializer(ModelSerializer):
+    class Meta:
+        model = Contents_table
+        fields = ['title', 'detail']
+
+
+class ContentsCreateAPI(APIView):
+    def get(self, request):
+        contents = Contents_table.objects.title
+
+        return_data = ContentsSerializer(contents, many=True).data
+
+        return Response(return_data)
+
+    def post(self, request):
+        contents = Contents_table.objects.create(
+            title=request.data['title'],
+            detail=request.data['detail'],
+            pos_x=request.data['pos_x'],
+            pos_y=request.data['pos_y'],
+        )
```

Comment:
serialziser 를 이용해서 is_valid 하는것도 추천 그리고 member_id, crew_id 도 request 에서 꺼내와야할 것 같음
```suggestion
        serializer = ContentsSerializer(data=request.data)
        serializer.is_valid(raise_exception=True)
        
        contents = Contents_table.objects.create(
            title=serializer.validated_data['title'],
            detail=serializer.validated_data['detail'],
            pos_x=serializer.validated_data['pos_x'],
            pos_y=serializer.validated_data['pos_y'],
            member_id=serializer.validated_data['member_id'],
            crew_id=serializer.validated_data['crew_id'],
        )
```